### ISSUE TITLE
CORS Issue After Rasa 1.2.4

### ISSUE THREAD

**Issue by dasxx179** [RasaHQ/rasa#4378]
**Rasa version**:
rasa                               1.2.7
rasa-core                          0.14.5
rasa-core-sdk                      0.14.0
rasa-nlu                           0.15.1
rasa-sdk                           1.2.0
**Rasa X version** (if used & relevant):
rasa-x                             0.20.2
**Python version**:
Python 3.7.3
python-engineio                    3.8.2.post1
python-socketio                    4.2.1
**Operating system** (windows, osx, ...):
MacOs
**Issue**:
Hi All,

I am attempting to use the mrbot-ai widget, and I notice that I do not have any issues when running rasa version 1.2.2 or 1.2.3.  However, specifically after version 1.2.4, I have noticed that I get the following error when starting my action server and running the following commands.  Note it displays "waiting for server" on the actual widget.  I have not modified anything aside from the version of rasa.  Any thoughts?
**Error (including full traceback)**:

<img width="547" alt="Screen Shot 2019-09-03 at 7 42 22 AM" src="https://user-images.githubusercontent.com/54422044/64174367-b5720500-ce1e-11e9-9559-41cafd1036bf.png">
<img width="843" alt="Screen Shot 2019-09-03 at 7 35 51 AM" src="https://user-images.githubusercontent.com/54422044/64174377-bb67e600-ce1e-11e9-8241-4d170afdfc18.png">

**Command or request that led to error**:
```
rasa run actions --actions demo.actions
rasa run -m models --enable-api --cors “*” --debug
```

**Content of configuration file (config.yml)** (if relevant):
```yml

```

**Content of domain file (domain.yml)** (if relevant):
```yml

```

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @amn41 will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by tmbo** [comment]
I think this should be fixed by #4389 now

**Comment by dasxx179** [comment]
I do not believe this is fixed.  I have updated to the versions that were specified by #4389 that you referenced, and I am getting a similar CORS issue.  
![image](https://user-images.githubusercontent.com/54422044/64562687-1f087c80-d313-11e9-9427-47b693e815c8.png)
